low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server.py

```diff
@@ -357,3 +381,190 @@ def get_nw_status(self, nw_interface, interface):
             nw_cable_conn_status = "UNKNOWN"
             # Log the error when logging class is in place.
         return nw_status, nw_cable_conn_status
+
+    def initialize_systemd(self):
+        """Initialize system bus and interface."""
+        self._bus = SystemBus()
+        self._systemd = self._bus.get_object(
+            SYSTEMD_BUS, "/org/freedesktop/systemd1")
+        self._manager = Interface(self._systemd, dbus_interface=MANAGER_IFACE)
+
+    @staticmethod
+    def initialize_cortx_build_info():
```

Comment:
This function can also be kept under Build() class as @cdeshmukh mentioned in above comment

Reply:
Moved function to `BuildInfo` class under low-level/framework/platforms/server/software.py